src/player_activity.h

```diff
@@ -29,10 +31,166 @@ class player;
 class translation;
 class activity_ptr;
 
+struct simple_task {
+    // Name of the target that's being processed
+    const std::string target_name;
+    /* Total number of moves required for this target/task to complete */
+    const int moves_total = 0;
+    /* The number of moves remaining for this target/task to complete */
+    int moves_left = 0;
+
+    inline const bool complete() const {
```

Comment:
```suggestion
    inline bool complete() const {
```
no need for a `const bool` here